Makefile

```diff
@@ -59,7 +59,9 @@ test-e2e-polybft:
     # We can not build with race because of a bug in boltdb dependency
 	go build -o artifacts/polygon-edge .
 	env EDGE_BINARY=${PWD}/artifacts/polygon-edge E2E_TESTS=true E2E_LOGS=true \
-	go test -v -timeout=1h30m ./e2e-polybft/e2e/...
+		go test -v -timeout 1h -run TestE2E_Bridge_ChangeVotingPower github.com/0xPolygon/polygon-edge/e2e-polybft/e2e
```

Comment:
Revert this to execute all e2e tests.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/ba933ad4577f451f5948be55c283e79ad11960c1